From d1010a14781d5aed3db0c4c03df414f9d4675014 Mon Sep 17 00:00:00 2001
From: dmeier <dmeier@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Tue, 11 Feb 2020 13:26:59 +0000
Subject: [PATCH 362/462] MANTIS 56312: Extend Desko-Penta Device-Handler by
 Accufill-SDK for reading US driving-licenses

setenv for the demo program

git-svn-id: svn://localhost/SelfServiceBuild/trunk@667 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/scripts/setenv.bat | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/SelfServiceBuild/Build/scripts/setenv.bat b/SelfServiceBuild/Build/scripts/setenv.bat
index 59bde79..2222f82 100644
--- a/SelfServiceBuild/Build/scripts/setenv.bat
+++ b/SelfServiceBuild/Build/scripts/setenv.bat
@@ -394,6 +394,10 @@ set DIR_SPEECH11=%DIR_GLOBALSLIB%\Microsoft_SDKs\Speech\v11.0
 set INC_SPEECH11=-I%DIR_SPEECH11%\Include
 set LIB_SPEECH11=%DIR_SPEECH11%\Lib\sapi.lib
 
+set DIR_ASSUREID=%DIR_GLOBALSLIB%\AssureID\sdk
+set INC_ASSUREID=-I%DIR_ASSUREID%\com -I%DIR_ASSUREID%\dotnet\4.0
+set LIB_ASSUREID=%DIR_ASSUREID%\dotnet\4.0\x86\AssureTec.AssureID.Platform.SDK.Extensions.lib
+
 set DIR_QTSDK=%DIR_GLOBALSLIB%\qtsdk\6.0\win32
 set INC_QTSDK=-I%DIR_QTSDK%\cincludes
 set LIB_QTSDK=%DIR_QTSDK%\libraries\qtmlclient.lib
-- 
2.41.0.windows.1

